### ISSUE TITLE
Error reading some JPG file

### ISSUE THREAD

**Issue by matianfu** [mscdex/mmmagic#119]
For some jpg file, the `detectFile` function returns an error, such as following:

```
Error: JPEG image data, Exif standard: [TIFF image data, big-endian, direntries=13, datetime=2017:11:01 18:24:35, GPS-Data, model=Nexus 6P, resolutionunit=2, height=0, yresolution=199, orientation=upper-left, software=angler-user 7.0 NBD90X 3254009 release-keys, xresolution=251, manufacturer=Huawei, width=0] name use count (30) exceeded
```

The problem is exactly the same as this bug report:

https://bugzilla.redhat.com/show_bug.cgi?id=1201630

The built-in file command (Ubuntu server amd64 16.04 LTS) won't emit such an error. So I guess your source code of libmagic should be updated.

**Comment by mscdex** [comment]
The version of libmagic currently in this repo is already the latest as of this writing (5.32) and that message still appears with the libmagic's master branch as of this writing. Unfortuately the [upstream bug tracker](https://bugs.gw.com/) is currently down so we will have to wait before this issue can be reported.

**Comment by matianfu** [comment]
Dear @mscdex , is it possible to change the upstream to libmagic source package of a distro, such as Ubuntu or Fedora. I found no problem for the built-in file command in those distros.

**Comment by mscdex** [comment]
Not really, right now I have to manually apply changes to upstream to get various things to work appropriately. I do not have actual patches for these changes at the moment.
